Source: Upstream
Upstream: Yes
Reason: Avoid build failures when ZSH sets $path

From 7c483431f1026e5bbfedc18c369652bb96f9f8dd Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?T=C3=B6r=C3=B6k=20Edwin?= <edwintorok@gmail.com>
Date: Sat, 28 Feb 2009 13:26:59 +0100
Subject: [PATCH] Do not add bogus directory arguments to link command lines.

* libltdl/config/ltmain.m4sh (func_mode_link): Ensure $path is
always initialized before it is used.  Reported for zsh, for
which $path contains $PATH entries even after "emulate sh", see
<http://bugs.debian.org/517501>.

Signed-off-by: Ralf Wildenhues <Ralf.Wildenhues@gmx.de>
---
 libltdl/config/ltmain.m4sh |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/libltdl/config/ltmain.m4sh b/libltdl/config/ltmain.m4sh
index 49e07c3..7fcf4cb 100644
--- a/libltdl/config/ltmain.m4sh
+++ b/libltdl/config/ltmain.m4sh
@@ -5624,6 +5624,7 @@ func_mode_link ()
 	  if test "$link_all_deplibs" != no; then
 	    # Add the search paths of all dependency libraries
 	    for deplib in $dependency_libs; do
+	      path=
 	      case $deplib in
 	      -L*) path="$deplib" ;;
 	      *.la)
-- 
1.6.4

